requirements.txt

```
@@ -0,0 +1,7 @@
+numpy
+pygame
+pytest
+tqdm
+pre-commit
+pygame
```

pygame in twice, `pre-commit` and `pytest` needed for contributing, but not running.

`tqdm` isn't used anywhere in the repo, but assume you want to use this `requirements.txt` to install into the repl with `pip install -r requirements.txt` after you `git clone`. In which case I'd keep it in.

```suggestion
pygame
tqdm
```

---

Yeah this is a broader point that we need a separate requirements for running and deving. will ticket this.